docs/onboardee-checklist.md

```diff
@@ -103,8 +103,5 @@ able to get back to them later upon need.
 * At this point, you're ready to ask your team members or your tutor for a dev task of sensible difficulty.
 * Ask your team members for feedback to see if they think you're doing fine and how to take it from here onwards.
 
-- are you able to get a review environment for your pull request?
-  - if not, raise the issue
-- are you able to diagnose why your build failed (if it did)?
```

Comment:
I'd find it relevant that the devs know where to see their builds and their logs.  But maybe that's for later?

Reply:
Agreed fully that they should know.. I personally think it's something to be learned as part of their daily work and not in the "bootstrap" phase of onboarding. Because we had discussed that the goal of the onboardee checklist is:

>  * The ultimate goal of this checklist is to get you from the moment you start working here to taking a dev ticket as smoothly as possible 🧡

And them not knowing where the builds status and logs are doesn't prevent them from taking a dev ticket I think.

Reply:
You might be right.